sw/device/silicon_creator/lib/boot_data.c

```diff
@@ -67,31 +67,90 @@ static rom_error_t boot_data_digest_compute(const void *boot_data,
                 "`digest` must be the first field of `boot_data_t`.");
 
   hmac_sha256_init();
```

Comment:
nit: maybe at this point we should have a one shot function as well for `hmac_sha256()`

Reply:
SG, created #10472.